package.json

```diff
@@ -9,7 +9,9 @@
     "start": "node server.js",
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "format": "prettier --write '**/*.{js,vue,scss,css,html}'",
```

Comment:
I'll look into this. We use `prettier-eslint` typically, so I want to make sure these commands do what I expect.

Reply:
`prettier-eslint` is not even in the `package.json` and `eslintrc`

Reply:
You're right! It was setup in our internal Vetur config, so we should probably amend our developer dependencies to include it. Then again, with Vue CLI 3 we might have to shake things up a bit anyways.